client/components/thank-you-v2/upsell/style.scss

```diff
@@ -81,4 +82,17 @@
 			color: var(--black-white-white, #fff);
 		}
 	}
+
+	.components-base-control__field {
+		margin-bottom: 0;
+	}
+	.components-select-control {
+		height: 36px;
+	}
+	.components-select-control__input {
+		font-size: $font-body-small !important;
```

Comment:
UPDATE: Oh, I see, it's so absurd bug with specificity - 3 duplications :(
```
.css-14s41nl-BackdropUI-backdropFocusedStyles.css-14s41nl-BackdropUI-backdropFocusedStyles.css-14s41nl-BackdropUI-backdropFocusedStyles {
....
}
```

I propose the next solution, which at least doesn't use `!important` and looks quite good and considers both my comments:

```
.components-select-control {
		height: 36px;

		.components-input-control__container {
			.components-select-control__input {
				font-size: $font-body-small;
			}
			.components-input-control__backdrop {
				border-color: var(--color-border-subtle);
			}
		}
	}
```

I hope that one day the build will be fixed and we will have one class in selectors. But for now I would go this way, instead of using `!important`, which is actually "workaround for the bug".

WDYT about it?

Reply:
> `!important` has a lot of downsides

In this particular case, I think I actually prefer `!important`, because:
1. The exact level of specificity that we have to override could theoretically change because of how little control we have over the `<SelectControl>` component.
2. The same thing goes for the structure of that component. There are no guarantees made regarding the structure.
3. No other components will override the styling from `client/components/thank-you-v2/upsell/style.scss`, so the risk of getting into a specificity war here seems low.